x-pack/plugins/fleet/public/applications/integrations/sections/epm/constants.tsx

```diff
@@ -89,6 +92,7 @@ export const AssetIcons: Record<KibanaAssetType, IconType> = {
   map: 'emsApp',
   lens: 'lensApp',
   security_rule: 'securityApp',
+  csp_rule_template: 'securityApp', // TODO ICON
```

Comment:
@kfirpeled, the mapping requires a new type to have an icon, even though this asset will not be presented in the UI (not needed in our use case) it's still required.